src/Models/Classes/ViewerConfigUtility.ts

```diff
@@ -663,7 +684,7 @@ abstract class ViewerConfigUtility {
 
         // Add behavior Ids from selected scene layers to Id dict
         config?.configuration.layers.forEach((layer) => {
-            if (selectedLayerIds.includes(layer.id)) {
+            if (selectedLayers.includes(layer.id)) {
                 layer.behaviorIDs.forEach((behaviorId) => {
                     uniqueBehaviorIds.set(behaviorId, '');
```

Comment:
unrelated to this PR but why do we set the value as an empty string here in the map? should this be a Map then? or should we set it to null or something?